File: nodeJS/app/models/feed.js
var Feed = function () {
	this.property('name', 'string', {required: true});

	this.belongsTo('User');
	this.hasMany('Tweets');
	this.hasMany('Subscribers');

	this.hasMany('Users', {through: "Subscribers"});
}

exports.Feed = Feed;

File: ASP.NET/src/4 - Core/SETwitter.Domain/Feed.cs

using System.Collections.Generic;

namespace SETwitter.Domain
{
    public class Feed
    {
        public long Id { get; set; }

        public string Name { get; set; }
        public virtual ICollection<Tweet> Tweets { get; set; }
        public virtual ICollection<User> Subscribers { get; set; }
        public User Owner { get; set; }
    }
}